subjects/mobile-dev/twenty-forty-eight/README.md

```diff
@@ -1,41 +1,33 @@
 ## twenty-forty-eight
 
-### Introduction:
+It is April 2014, and the popular game "2048" has not yet been released. Your task is to implement the game before the official release. The objective of the game is to slide numbered tiles on a grid to combine them and create a tile with the number 2048.
 
-It is April 2014, and the popular game "2048" has not yet been released. Your task is to implement the game before the official release. To do this, you will need to create a 4x4 table and fill it with tiles valued at either 2 or 4. The game should start with 3-4 tiles placed randomly on the board.
-
-Players should be able to swipe the tiles in any of the four directions, and the tiles will move as far as possible in that direction. If two tiles with the same value collide, they should merge into one tile with twice the value, and the score should be updated accordingly. After each move, a new tile should appear randomly in an empty slot on the board.
-
-Use any [animation](https://docs.flutter.dev/development/ui/widgets/animation) to smoothly move the tiles. You can choose the colors for the tiles, but it is preferred to use the colors from the original "2048" game.
-
-The game should stop when no legal moves are possible and allow the user to restart. When the game is over, the best score should be updated if necessary.
+<center>
+<img src="./resources/2048.01.png?raw=true" style = "width: 500px !important; height: 300px !important;"/>
```

Comment:
The image looked a bit squeezed - the original size is w:228, h:328

Reply:
done